tests/drogon/test_well_rate.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+import pytest
+from everest.bin.everest_script import everest_entry
+
+
+def test_well_rate_short(capsys):
+    """
+    Run a modified Drogon well_rate tutorial test case.
+
+    Modify to run flow sims for first 10 realizations only and
+    reduce the max_batch_num to 2 for faster testing.
+    """
+
+    config_path = Path("data/drogon/well_rate/everest/model/wellrate_experiment.yml")
+    config_path.write_text(
+        config_path.read_text()
+        .replace("max_batch_num: 10", "max_batch_num: 2")
+        .replace("realizations: r{{range(100) | list()}}", "realizations: 0-9")
+        .replace("name: lsf", "name: lsf\n    lsf_queue: test")
+    )
+
+    try:
+        everest_entry([str(config_path), "--skip-prompt"])
+    except SystemExit as e:
```

Comment:
is this capture worth it? If we just to everest_entry with no `try` it would be similarly simple to debug?